setup.cfg

```diff
@@ -0,0 +1,39 @@
+[metadata]
+name = gateio_new_coins_announcements_bot
```

Comment:
'gateio_new_coins_announcements_bot' this is the internal module name. 
I wasn't sure what to name it, the repo name is too long IMO.

Reply:
- [ ] Is this name fine?